<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FoaDetectWidget</class>
 <widget class="QWidget" name="FoaDetectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_build">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>15</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #B0B0B0;  /* 浅灰色 */
    color: white;               /* 白色文本 */
    border-radius: 10px;        /* 圆角边框 */
    padding: 5px 15px;          /* 内边距 */
}

QPushButton:hover {
    background-color: #A0A0A0;  /* 鼠标悬停时的颜色 */
}

QPushButton:pressed {
    background-color: #909090;  /* 按下时的颜色 */
}
</string>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>890</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>显示区</string>
   </property>
   <widget class="QListWidget" name="listWidget_processed">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>40</y>
      <width>591</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="DragListWidget" name="listWidget_pending">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>40</y>
      <width>231</width>
      <height>311</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FOA待检列表</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_emptyPending">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>18</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    QPushButton {
        background-color: red;        /* 背景色设为红色，警示效果 */
        color: white;                 /* 文字颜色设为白色 */
        border: 2px solid #cc0000;    /* 边框颜色稍深，突出按钮 */
        border-radius: 10px;          /* 圆角边框 */
        padding: 1px;                 /* 减少内边距，以便显示完整文本 */
    }
    QPushButton:hover {
        background-color: #ff6666;    /* 鼠标悬停时背景色变浅 */
        border: 2px solid #ff3333;    /* 悬停时边框颜色变浅 */
    }
    QPushButton:pressed {
        background-color: #cc0000;    /* 按下时背景色稍深 */
        border: 2px solid #990000;    /* 按下时边框颜色更深 */
    }</string>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dropTips">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>从系统资源管理器中
拖拽资源至此</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>不合规的FOA文件列表</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;在此输入待检测重复内容的文件路径&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;在此输入待检测重复内容的文件路径&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="title">
    <string>In</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_inPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>输入路径</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_inPath">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DragListWidget</class>
   <extends>QListWidget</extends>
   <header>sample.qt.src.pyui.DragListWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
